python/dgl/distributed/client.py

```diff
@@ -2,33 +2,155 @@
 
 """Functions used by client."""
```

Comment:
i think the file should be called 'rpc_client.py'.

Reply:
Ok, use `rpc_client` and `rpc_server`